### DISCUSSION TITLE
Suggestion: Automatically create "ungrouped" list

### DISCUSSION THREAD

**Discussion by bradlitterell** [community/community#20886] [Other Feature Feedback, Questions, & Ideas]
The lists feature for starred repositories looks super useful, thank you!

However, I noticed when trying to break up a large list of starred repositories, it is difficult to keep track of which have been assigned to lists and which have not.

It would be helpful if there were an automatically curated "ungrouped/unassigned" list - Stars that are not assigned to any other list.

**Comment by jfullstackdev** [comment]
how would that be difficult to track ? 

if I will be grouping it again, then I will see 

![image](https://user-images.githubusercontent.com/47092464/180001827-89a4c7bd-265d-4be0-8c41-e92e90eefa8d.png)

you will know it does not belong to any group because they are all unchecked

**Reply by bradlitterell** [reply]
Yeah, but that requires clicking on the down-arrow to display the list-of-lists.  Suppose you are trying to sort 100 starred repositories.  Clicking down-arrow on all of them to find the 2 that are uncategorized doesn't scale.

**Comment by Stuntman58** [comment]
https://github.com/orgs/github-community/discussions/20886

**Reply by Stuntman58** [reply]
Don't shoot the messenger because I really don't know what I'm doing here but I'm trying to learn

**Reply by bradlitterell** [reply]
? puzzled - comment points back to this thread?
